### ISSUE TITLE
smerge should open the current folder if no command line arguments are specified

### ISSUE THREAD

**Issue by sxenos** [sublimehq/sublime_merge#1620]
**Problem description**

When running sublime merge on the command line, typing "smerge" on its own opens the sublime merge GUI but does not open any particular git repository.

I'm constantly assuming it will behave like all git commands and act on the repository in the current folder by default. "smerge ." does what I want, but it's hard to remember to type in a path when this differs from other git commands.

**Preferred solution**

smerge with no arguments should open the git repository in the current folder, if one exists.

**Alternatives**

Keep the smerge executable as it is and include an alternative command-line launcher that behaves more like other git commands.

**Comment by srbs** [comment]
I don't think this should happen as the current behavior is non-destructive. If it was treated as equivalent to `smerge .`, then the existing session would be dropped in favor of the current directory. I doubt any existing cli user would want or expect that change in behavior.

**Comment by sxenos** [comment]
Perhaps it could it start a new session in the current directory, or bring an existing one to the foreground if it was already open.
